### PR TITLE
Write JSONIX Service tests

### PR DESCRIPTION
**Pull request by oehm-smith**

### CODE REVIEW

File: src/client/app/shared/jsonix/jsonix.service.spec.ts [hunk 1]
```diff
@@ -0,0 +1,69 @@
+import { ReflectiveInjector } from '@angular/core';
+
+import { JsonixService } from './jsonix.service';
+
+export function main() {
+  describe('Jsonix Service', () => {
+    let jsonIxService: JsonixService;
```
Comment (lbodor) [inline src/client/app/shared/jsonix/jsonix.service.spec.ts]: Spelling, jsonIx should be jsonix.

File: src/client/app/shared/jsonix/jsonix.service.spec.ts [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+ xmlns:geo="urn:xml-gov-au:icsm:egeodesy:0.3"
+ xmlns:gml="http://www.opengis.net/gml/3.2"
+ xmlns:ns9="http://www.w3.org/1999/xlink"
+ xmlns:gmd="http://www.isotc211.org/2005/gmd"
+ xmlns:gmx="http://www.isotc211.org/2005/gmx" 
+ xmlns:om="http://www.opengis.net/om/2.0" xmlns:gco="http://www.isotc211.org/2005/gco" 
+ xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" gml:id="GeodesyMLType_20">
+<geo:siteLog></geo:siteLog>
+</geo:GeodesyML>`;
+      let json: string = jsonIxService.geodesyMlToJson(geodesyml);
```
Comment (lbodor) [inline src/client/app/shared/jsonix/jsonix.service.spec.ts]: Spelling, geodesyMltoJson should be geodesyMLToJson.

File: src/client/app/shared/jsonix/jsonix.service.spec.ts [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    beforeEach(() => {
+
+      let injector = ReflectiveInjector.resolveAndCreate([
+        JsonixService,
+      ]);
+      jsonIxService = injector.get(JsonixService);
+    });
+
+    it('should be defined', () => {
+      expect(JsonixService.WS_URL).not.toBeNull();
```
Comment (lbodor) [inline src/client/app/shared/jsonix/jsonix.service.spec.ts]: Why does the jsonix service need WS_URL?

File: src/client/app/shared/jsonix/jsonix.service.spec.ts (hunk 1 above)
Comment (oehm-smith) [inline src/client/app/shared/jsonix/jsonix.service.spec.ts]: IntelliJ sees jsonix as mispelled and its annoying.

File: src/client/app/shared/jsonix/jsonix.service.spec.ts (hunk 3 above)
Comment (oehm-smith) [inline src/client/app/shared/jsonix/jsonix.service.spec.ts]: Good point!

File: src/client/app/shared/jsonix/jsonix.service.spec.ts (hunk 1 above)
Comment (lbodor) [inline src/client/app/shared/jsonix/jsonix.service.spec.ts]: I'll change it, vi doesn't mind :)

**Pull request merged**